python/py_defines.h

```diff
@@ -50,4 +50,7 @@ Numeric decoder derived from from TCL library
 
 #define PyString_FromString     PyUnicode_FromString
 
+#define PyUnicode_AsUTF8String(o) \
+    (PyUnicode_AsEncodedString((o), "utf-8", "surrogatepass"))
+
```

Comment:
This code seems unused?

If you're aiming for `surrogatepass` as a generic solution, it's a recipy to produce invalid UTF-8:

```
In [6]: '\udcff'.encode('utf-8', 'surrogatepass').decode('utf-8')                                                                                      
[..]
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xed in position 0: invalid continuation byte
```

Are you aware?